src/elements/form-field/form-field/form-field.scss

```diff
@@ -405,7 +405,7 @@
 
 // Input
 
-.sbb-form-field__input ::slotted(:where(input, select, textarea, sbb-select)) {
+.sbb-form-field__input ::slotted(:where(input, select, textarea, sbb-select, sbb-date-input)) {
```

Comment:
@TomMenga little notice to you, that after merge your chip input PR needs an update regarding this :)